.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "xbmc/addons/kodi-addon-dev-kit"]
+	path = xbmc/addons/kodi-addon-dev-kit
+	url = git://github.com/AlwinEsch/kodi-addon-dev-kit.git
```

Comment:
You mean as `externalproject` to build the dev-kit during Jenkins and Travis build? If yes then it is easy with `externalproject` to define to a folder inside Kodi.

But I see the Problem if the add-on is complete created alone without the build on Kodi. There is a way to clone only a part from repository, but the download use the full code inside them and is a bit to long for a add-on build. Also need this way a rework by hand or script. Has found it here http://stackoverflow.com/questions/600079/is-there-any-way-to-clone-a-git-repositorys-sub-directory-only/13738951#13738951 but inacceptable for me.